CSIRO BSD MIT License v2.0.txt

```diff
@@ -0,0 +1,43 @@
+CSIRO Open Source Software License Agreement (variation of the BSD / MIT License)
```

Comment:
Would you be willing to license your contributions under the BSD license that IPython uses? We do not want to have different licenses applying to different parts of IPython.

This file also appears to be full of unicode errors.